packages/client-core/src/components/World/LocationLoadHelper.tsx

```diff
@@ -98,6 +98,10 @@ const injectedSystems: SystemModuleType<any>[] = [
   {
     type: 'PRE_RENDER',
     systemModulePromise: import('@xrengine/client-core/src/systems/AvatarUISystem')
+  },
+  {
+    type: 'PRE_RENDER',
+    systemModulePromise: import('@xrengine/engine/src/networking/systems/MediaStreamSystem')
```

Comment:
we should add this back to initializeRealitmeSystems in initialiseEngine.ts along with the conditional argument